R/gene_to_region.R

```diff
@@ -9,7 +9,7 @@
 #'
 #' @param gene_symbol A vector of one or more gene symbols.
 #' @param ensembl_id A vector of one or more Ensembl IDs.
-#' @param genome_build Reference genome build. Possible values are "grch37" (default) or "hg38".
+#' @param projection Reference genome build. Possible values are "grch37" (default) or "hg38".
```

Comment:
The renaming of this parameter, does that affect the rest of the code base? I.e are there examples or internal calls of `gene_to_region` that have to be updated to call this new parameter name? I ran my function to look for internal calls and this is what it returned, not sure if the projection/genome_build parameter is called on these lines though.

```
> look_for_a_string(to_check = "gene_to_region(")

GAMBLR.results/R/get_cn_states.R:27
GAMBLR.results/R/get_cnv_and_ssm_status.R:130
GAMBLR.utils/R/count_ssm_by_region.R:20
GAMBLR.utils/R/region_to_gene.R:5
GAMBLR.utils/R/region_to_gene.R:19
GAMBLR.utils/R/gene_to_region.R:22
GAMBLR.utils/R/gene_to_region.R:25
GAMBLR.viz/R/fancy_circos_plot.R:11
GAMBLR.viz/R/fancy_circos_plot.R:39
GAMBLR.viz/R/fancy_ideogram.R:86
GAMBLR.viz/R/focal_cn_plot.R:30
GAMBLR.viz/R/focal_cn_plot.R:50
```

Reply:
right, those will definitely need to be updated and handled elsewhere in the appropriate PRs for other children repos. The 5 matches inside of this repo are some that need code updates (4/5, already handled in this PR) and those that do not need code update (1/5, example of unrelated function that also calls it without offending genome_build/projection arguments) so all is good on the utils side